### DISCUSSION TITLE
SnowflurryPlots installation problem

### DISCUSSION THREAD

**Discussion by malvi-rp** [anyonlabs/Snowflake.jl#301] [Q&A]
Hi all,

I am trying to use SnowflurryPlots for my program, but I get this error. Can you assist me with this?

-----------------------------------------
Updating git-repo `[https://github.com/SnowflurrySDK/SnowflurryPlots.jl`](https://github.com/SnowflurrySDK/SnowflurryPlots.jl%60)
   Resolving package versions...
Unsatisfiable requirements detected for package WebSockets [104b5d7c]:
 WebSockets [104b5d7c] log:
 ├─possible versions are: 1.0.0-1.6.0 or uninstalled
 ├─restricted by compatibility requirements with HTTP [cd3eb016] to versions: 1.6.0 or uninstalled
 │ └─HTTP [cd3eb016] log:
 │   ├─possible versions are: 0.6.10-1.9.14 or uninstalled
 │   └─restricted to versions 1.9.6-1 by Snowflurry [7bd9edc1], leaving only versions 1.9.6-1.9.14
 │     └─Snowflurry [7bd9edc1] log:
 │       ├─possible versions are: 0.1.0 or uninstalled
 │       └─Snowflurry [7bd9edc1] is fixed to version 0.1.0
 ├─restricted by julia compatibility requirements to versions: 1.0.0-1.5.9 or uninstalled, leaving only versions: uninstalled
 └─restricted by compatibility requirements with WebIO [0f1e0344] to versions: 1.0.0-1.6.0 — no versions left
   └─WebIO [0f1e0344] log:
     ├─possible versions are: 0.2.5-0.8.21 or uninstalled
     ├─restricted by compatibility requirements with PlotlyJS [f0f68f2c] to versions: 0.2.5-0.8.21
     │ └─PlotlyJS [f0f68f2c] log:
     │   ├─possible versions are: 0.11.0-0.18.10 or uninstalled
     │   ├─restricted to versions * by SnowflurryPlots [fa4a7b36], leaving only versions 0.11.0-0.18.10
     │   │ └─SnowflurryPlots [fa4a7b36] log:
     │   │   ├─possible versions are: 0.1.0 or uninstalled
     │   │   └─SnowflurryPlots [fa4a7b36] is fixed to version 0.1.0
     │   └─restricted by compatibility requirements with Requires [ae029012] to versions: 0.12.4-0.18.10 or uninstalled, leaving only versions: 0.12.4-0.18.10
     │     └─Requires [ae029012] log:
     │       ├─possible versions are: 0.5.0-1.3.0 or uninstalled
     │       └─restricted by compatibility requirements with Plots [91a5bcdd] to versions: 1.0.0-1.3.0
     │         └─Plots [91a5bcdd] log:
     │           ├─possible versions are: 0.12.1-1.39.0 or uninstalled
     │           └─restricted to versions 1.25.0-1 by SnowflurryPlots [fa4a7b36], leaving only versions 1.25.0-1.39.0
     │             └─SnowflurryPlots [fa4a7b36] log: see above
     └─restricted by compatibility requirements with Requires [ae029012] to versions: 0.8.3-0.8.21 or uninstalled, leaving only versions: 0.8.3-0.8.21
       └─Requires [ae029012] log: see above

**Comment by plerouxanyon** [comment]
Hi,

I have tried reproducing your error, but Snowflurry and SnowflurryPlots seem to work as expected.

In a fresh Julia 1.9.3 project, the following works.

```julia
import Pkg
Pkg.add(url="https://github.com/SnowflurrySDK/Snowflurry.jl", rev="main")
Pkg.add(url="https://github.com/SnowflurrySDK/SnowflurryPlots.jl", rev="main")
Pkg.add("WebSockets")
```

We want to get you up and running with Snowflurry and SnowflurryPlots, but to look at this further, we need to know your Julia version and what dependencies you have in your `Project.toml` file.
